inference/inference-tfjs/src/jsMain/kotlin/io.kinference.tfjs/operators/ml/ZipMap.kt

```diff
@@ -0,0 +1,116 @@
+package io.kinference.tfjs.operators.ml
+
+import io.kinference.attribute.Attribute
+import io.kinference.data.ONNXData
+import io.kinference.graph.Contexts
+import io.kinference.ndarray.arrays.NDArrayTFJS
+import io.kinference.ndarray.arrays.NumberNDArrayTFJS
+import io.kinference.ndarray.extensions.dataFloat
+import io.kinference.operator.*
+import io.kinference.protobuf.message.AttributeProto
+import io.kinference.protobuf.message.TensorProto
+import io.kinference.tfjs.TFJSData
+import io.kinference.tfjs.data.map.TFJSMap
+import io.kinference.tfjs.data.seq.TFJSSequence
+import io.kinference.tfjs.data.tensors.TFJSTensor
+import io.kinference.tfjs.data.tensors.asTensor
+import io.kinference.types.TensorShape
+import io.kinference.types.ValueTypeInfo
+
+sealed class ZipMap(
+    name: String,
+    info: OperatorInfo,
+    attributes: Map<String, Attribute<Any>>,
+    inputs: List<String>,
+    outputs: List<String>
+) : Operator<TFJSTensor, TFJSSequence>(name, info, attributes, inputs, outputs) {
+    companion object {
+        private val DEFAULT_VERSION = VersionInfo(sinceVersion = 1)
+
+        operator fun invoke(name: String, version: Int?, attributes: Map<String, Attribute<Any>>, inputs: List<String>, outputs: List<String>): ZipMap {
+            return when (version ?: DEFAULT_VERSION.sinceVersion) {
+                in ZipMapVer1.VERSION.asRange() -> ZipMapVer1(name, attributes, inputs, outputs)
+                else -> error("Unsupported version of ZipMap operator: $version")
+            }
+        }
+    }
+}
+
+
+class ZipMapVer1(
+    name: String,
+    attributes: Map<String, Attribute<Any>>,
+    inputs: List<String>,
+    outputs: List<String>
+) : ZipMap(name, INFO, attributes, inputs, outputs) {
+    companion object {
+        private val OUT_TYPE_CONSTRAINTS = setOf(TensorProto.DataType.INT64, TensorProto.DataType.STRING, TensorProto.DataType.FLOAT)
+
+        private val INPUTS_INFO = listOf(
+            IOInfo(0, setOf(TensorProto.DataType.FLOAT), "X", optional = false)
+        )
+
+        private val OUTPUTS_INFO = listOf(
+            IOInfo(0, OUT_TYPE_CONSTRAINTS, "Z", optional = false)
+        )
+
+        private val ATTRIBUTES_INFO = listOf(
+            AttributeInfo("classlabels_int64s", setOf(AttributeProto.AttributeType.INTS), required = false),
+            AttributeInfo("classlabels_strings", setOf(AttributeProto.AttributeType.STRINGS), required = false)
+        )
+
+        internal val VERSION = VersionInfo(sinceVersion = 1)
+        private val INFO = OperatorInfo("ZipMap", ATTRIBUTES_INFO, INPUTS_INFO, OUTPUTS_INFO, VERSION, domain = "ai.onnx.ml")
```

Comment:
Maybe we can move standard domains somewhere? I think `"ai.onnx.ml"` is used not once.